app/routes/task_routes.py

```diff
@@ -1 +1,71 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, abort
+from datetime import datetime
+from os import environ
+import requests
+
+from app.models.task import Task
+from app.db import db
+from app.routes.route_utilities import *
+
+bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+@bp.post("/", strict_slashes=False)
+def create_task():
+    return create_class_instance(Task, request, ["title", "description"])
```

Comment:
Same comment as in `goal_routes.py`: looks like `create_class_instance` never accesses the last param `required_fields` so it should be removed

```suggestion
    return create_class_instance(Task, request)
```